### PR TITLE
Logout user

### PR DESCRIPTION
**Pull request by tmccoy529**
Added logout ability for users that are not the current user. Started the process of converting the set field values to a utility. Service Request Form has been converted.

### CODE REVIEW

File: src/components/ServiceRequestForm.jsx [hunk 1]
```diff
@@ -17,7 +16,7 @@ import OtherAnimalsTypes from "./otherAnimalTypes";
 import SexType from './sexType';
 import AnimalColorType from './animalColorType';
 import AnimalBreedType from './animalBreedType';
-import { URLRouting } from '../utilities/FormHelpers';
+import { URLRouting, setFieldValues } from '../utilities/FormHelpers';
```
Comment (martypowell) [inline src/components/ServiceRequestForm.jsx]: We don't have a standards document so i can't block for this, but in these standalone js files the expecation is the function name is uppercase `SetFieldNames`. That way we know its coming from somewhere else.

File: src/components/ServiceRequestForm.jsx [hunk 2]
```diff
@@ -124,18 +123,20 @@ const ServiceRequestForm = (props, errors, touched) => {
...
+					RequestPage: resultFormFieldNames.data.RequestPage,
+					MapPage: resultFormFieldNames.data.MapPage,
+					AdditionalInfoPage: resultFormFieldNames.data.AdditionalInfoPage,
+					SignInPage: resultFormFieldNames.data.SignInPage,
+					SignUpPage: resultFormFieldNames.data.SignUpPage,
+					ResetPasswordPage: resultFormFieldNames.data.ResetPasswordPage,
+					ContactID: contactID,
+					requestType: selectedType(),
+					subRequestType: selectedSubType()
+				}
```
Comment (martypowell) [inline src/components/ServiceRequestForm.jsx]: please add semi colon.

File: src/components/ServiceRequestForm.jsx (hunk 1 above)
Comment (tmccoy529) [reply src/components/ServiceRequestForm.jsx]: Changed and pushed

File: src/components/ServiceRequestForm.jsx (hunk 2 above)
Comment (tmccoy529) [reply src/components/ServiceRequestForm.jsx]: added

**Pull request merged**